app/dashboard/templates/dashboard/sidebar_search_hackathon.html

```diff
@@ -51,10 +51,10 @@
 
             
         {% for org in orgs %}
-          <div class="form__radio option {{org}}">
-            <input name="org" id="{{org}}" type="radio" value="{{org}}" val-ui="Turing-Chain">
```

Comment:
what the heck is "Turing-Chain" ? 🤔

Reply:
I have no clue :P I was tempted to remove it but then thought maybe check it out later :P

Reply:
😂